frontend/src/components/secure-message/MessageDisplay.vue

```diff
@@ -507,16 +508,35 @@ export default {
         });
       router.push({name: `exchange_inbox_${this.secureExchange.ministryOwnershipGroupRoleIdentifier}`});
     },
+    isClaimable(){
+      if(this.secureExchange.reviewer === ''){
+        return true;
+      }
+      if(this.secureExchange.reviewer !== '' && this.secureExchange.reviewer !== this.userInfo.userName){
+        return true;
+      } else {
+        return false;
+      }
```

Comment:
Is there a way to simplify the flow for this? 
I'm thinking like a stepwise check 
Line 512 we check whether or not a reviewer is there. If not the message is always claimable. 

Line 515 If the reviewer is not the same as user. Then return true 

Then in all other cases it's false 

I'm thinking something like this 
 isClaimable(){
      if(this.secureExchange.reviewer === ''){
        return true;
      }
      if(this.secureExchange.reviewer !== this.userInfo.userName){
        return true;
      } 
        return false;
    

^^^ removes if else statement

Reply:
@SoLetsDev You can simplify further:

`isClaimable(){
  return this.secureExchange.reviewer === '';
}`

Reply:
> @SoLetsDev You can simplify further:
> 
> `isClaimable(){ return this.secureExchange.reviewer === ''; }`

But the user needs to be able to claim it even when someone else has already. According to the ticket.

Reply:
@TimOsse - No worries, we can just make it a double condition like this:

`
isClaimable(){ return this.secureExchange.reviewer === '' && this.secureExchange.reviewer !== this.userInfo.userName; }
`